<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaiterView</class>
 <widget class="QDialog" name="WaiterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>925</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>925</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WaiterView</string>
  </property>
  <widget class="QGroupBox" name="gbTodo">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>225</y>
     <width>525</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string>当前任务清单</string>
   </property>
   <widget class="QListWidget" name="lstPlate">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>75</y>
      <width>225</width>
      <height>325</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QListWidget" name="lstDemand">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>75</y>
      <width>225</width>
      <height>325</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblDemand">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>35</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>顾客需要的服务</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlate">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>35</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>已做好的菜品</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnServe">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>425</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>派送选定菜品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDemand">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>425</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>完成选定服务</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblWaiterBGN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbDone">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>175</y>
     <width>325</width>
     <height>550</height>
    </rect>
   </property>
   <property name="title">
    <string>已完成的任务</string>
   </property>
   <widget class="QTableView" name="tbDone">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>275</width>
      <height>500</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>175</y>
     <width>350</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProfile">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>175</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>个人资料</string>
   </property>
  </widget>
  <zorder>lblWaiterBGN</zorder>
  <zorder>gbTodo</zorder>
  <zorder>gbDone</zorder>
  <zorder>lblInfo</zorder>
  <zorder>btnProfile</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnProfile</sender>
   <signal>clicked()</signal>
   <receiver>WaiterView</receiver>
   <slot>OnProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnServe</sender>
   <signal>clicked()</signal>
   <receiver>WaiterView</receiver>
   <slot>OnDeliver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>672</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDemand</sender>
   <signal>clicked()</signal>
   <receiver>WaiterView</receiver>
   <slot>OnDemand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>672</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstPlate</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>WaiterView</receiver>
   <slot>OnSelPlate(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstDemand</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>WaiterView</receiver>
   <slot>OnSelDemand(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnProfile()</slot>
  <slot>OnDeliver()</slot>
  <slot>OnDemand()</slot>
  <slot>OnSelPlate(int)</slot>
  <slot>OnSelDemand(int)</slot>
 </slots>
</ui>
